### ISSUE TITLE
Switch CalendarLayout in runtime

### ISSUE THREAD

**Issue by marcustindab** [yurkinh/Plugin.Maui.Calendar#41]
Hi,

First and foremost - great job with the plugin!

Is it possible to switch WeekLayout at runtime, or is this not a tested/verified feature?

For example; switching between week or month views.

**Comment by yurkinh** [comment]
Hi @marcustindab 
Thanks for the kind words.
Currently, this functionality is not available but I plan to  add this in the next major version (2.0.0)

**Comment by yurkinh** [comment]
@marcustindab 
Check please CalendarLayout property I think that's what you are looking for)

**Comment by marcustindab** [comment]
Yeah, I tried to use that property but it did not work properly... The rendering of the view lagged behind. Perhaps I need to make further adjustments?

Also another question on the topic; is it possible to bind another Event-source to the calendar, besides the EventCollection? I would like to use the same structure as decribed in the .Net Maui Grouped Collection documentation, like the following example:

```
public class AnimalGroup : List<Animal>
{
    public DateTime Date { get; private set; }

    public AnimalGroup(DateTime date, List<Animal> animals) : base(animals)
    {
        Date = date;
    }
}
```

**Comment by yurkinh** [comment]
Hi @marcustindab 
Check please latest v1.1.6
